packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -429,6 +429,42 @@ class StackViewport extends Viewport {
     colormap: CPUFallbackColormapData | ColormapPublic
   ) => void;
 
+  /**
+   * Sets the sharpening for the current viewport.
+   * @param sharpening - The sharpening configuration to use.
+   */
+  private setSharpening = (sharpening: {
+    enabled: boolean;
+    intensity?: number;
+  }): void => {
+    // Store sharpening settings directly on the class
+    this.sharpening = sharpening;
+
+    this.render();
+  };
+
+  /**
+   * Get render passes for this viewport.
+   * If sharpening is enabled, returns appropriate render passes.
+   * @returns Array of VTK render passes or null if no custom passes are needed
+   */
+  public getRenderPasses = () => {
```

Comment:
can this be private maybe?

Reply:
this can't be private, the rendering engine needs to call it, it's like getViewportWidgets